wpengine/faustjs
# MERL-1185: Redirect Off Doesn't Rewrite Post Preview Link

## Description
Updated post preview link to only rewrite if enable redirects is off.

## Testing
1. Setup a WP instance with Faust
2. Ensure the FaustWP Plugin is linked or installed correctly
3. In Faust Plugin, make sure `Enable public route redirects` is unchecked
4. Clicking preview in a post should work properly using Wordpress default post preview link
